accepted/2020/workloads/workload-manifest.md

```diff
@@ -0,0 +1,430 @@
+
+## Template Packs
+
+Template packs are NuGet packages to be used by the dotnet templating engine. When a template pack is installed installed, the nupkg package file is placed in `dotnet/template-packs/` but is *not* extracted. Visual Studio and `dotnet new` are expected to automatically detect when templates nupkgs are installed and uninstalled to this folder and update their template hives automatically.
+
+
+## Tools Packs
+
+Tools packs are .NET global tools packages. Tools packs are installed to `dotnet/tools-packs/{package-id}/{package-version}/` and can be invoked via `dotnet` e.g. invoking `dotnet foo`, the runtime will use global.json and the manifest to determine the appropriate version of `dotnet-foo` to run.
+
+Tools packs are intended to be used for tools that are exposed part of the development experience. Tools that are only expected to be invoked by build targets should be distributed in an SDK pack.
+
+# Manifest Files
+
+## Composition
+
+Multiple manifests may be present in a .NET SDK band. Each manifest has its own ID and they are versioned independently. Components and packs may not be duplicated in multiple manifests within a band, though manifests may reference components and packs in other manifests that are expected to be installed.
+
+Manifests are not an arbitrary extensibility mechanism; they are intended to provide some flexibility in the distribution and composition of a coherent whole. For example, there could be a *dotnet* manifest used to update in-box shared runtimes, a *xamarin-ios* manifest used to describe and update the Xamarin.iOS optional components, and a *uwp* manifest used to describe and update the UWP optional components. These could all be produced from separate repositories using separate build and release processes.
+
+At any given point in time, the latest available version of all the manifests for an SDK series must always be able to be combined into a single consistent whole (in memory, not persisted to disk).
+
+Each manifest may have a `WorkloadManifest.targets` MSBuild file beside it that is unconditionally imported into all projects. This targets file contains conditioned imports to add automatic referenced to SDK packs based on project properties, and its usage and recommended patterns are described in more detail in the [workload resolvers spec](/workload-resolvers.md).
+
+## Packaging
+
+Manifests are packaged and distributed as NuGet packages. They will use the existing package type called `DotnetPlatform` so they cannot be accidentally used as a `PackageReference`. The manifest’s `sdk-manifest.json` and `WorkloadTargets.targets`  are in the root folder of the NuGet. The ID of the NuGet is `{manifest-id}-{sdk-band}` and the version of the NuGet is the manifest version.
+
+As manifests are distributed as NuGets, it’s possible to use preview versions of workloads by subscribing to preview feeds.
```

Comment:
Good scenario.

How would this work? Would you imagine you install a preview SDK, and it will look for a manifest package for a given workload in that same band (across registered feeds)? What would you expect if it doesn't find one?

Reply:
I meant that you could install previews of a workload into a stable SDK band by subscribing to a preview feed that contained manifest updates for that band that contained preview packs. Clarified.